CHANGELOG.md

```diff
@@ -16,9 +16,9 @@
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
 ### BREAKING CHANGES
 - Rename `context` prop to `mountNode` in `PortalInner` @layershifter ([#1288](https://github.com/stardust-ui/react/pull/1288))
+- Add focus border styling mechanism @Bugaa92 ([#1269](https://github.com/stardust-ui/react/pull/1269))
```

Comment:
Add `in Teams theme` in the changelog entry

Reply:
done, thanks